dbms/src/Common/BackgroundTask.cpp

```diff
@@ -74,13 +73,17 @@ void CollectProcInfoBackgroundTask::begin()
 void CollectProcInfoBackgroundTask::memCheckJob()
 {
     double resident_set;
+    Int64 cur_proc_num_threads;
+    UInt64 cur_virt_size;
     while (!end_syn)
     {
-        process_mem_usage(resident_set);
+        process_mem_usage(resident_set, cur_proc_num_threads, cur_virt_size);
         resident_set *= 1024; // unit: byte
         real_rss = static_cast<Int64>(resident_set);
-
-        usleep(100000); // sleep 100ms
+        proc_num_threads = cur_proc_num_threads;
+        proc_virt_size = cur_virt_size;
+        baseline_of_query_mem_tracker = root_of_query_mem_trackers->get();
```

Comment:
Assign a pointer to Int64?

Reply:
it's `root_of_query_mem_trackers->get() ` not `root_of_query_mem_trackers.get()`. `root_of_query_mem_trackers->get()` get the amount of mem tracker.